scripts/embargo_registrations.py

```diff
@@ -46,7 +46,8 @@ def main(dry_run=True):
                         parent_registration.registered_from.add_log(
                             action=NodeLog.EMBARGO_APPROVED,
                             params={
-                                'node': parent_registration._id,
+                                'node': parent_registration.registered_from_id,
+                                'registration': parent_registration._id,
```

Comment:
I'm not certain that it needs to be removed -- it could be useful to store information about which registration this log refers to, even if it's not displayed currently.